src/ng-select/lib/ng-select.component.ts

```diff
@@ -191,6 +191,13 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
     private _onChange = (_: any) => { };
     private _onTouched = () => { };
 
+    /**
+     * Is IME in compostion mode
+     * 
+     * @memberof NgSelectComponent
+     */
+    private _IMEComposition = false;
```

Comment:
`IME` abbreviation is not very common so it adds more confusion to variable name than information, I would suggest naming it simpler like `_isComposing` or `_composing`. Also, remove JSDoc comments as other members don't have them.

Reply:
Fixed in ab098331563ac33f6e50204f9088a491c7742493